Mathlib/Topology/Separation.lean

```diff
@@ -60,8 +60,8 @@ This file defines the predicate `SeparatedNhds`, and common separation axioms
   these results are part of the typeclass inference system (e.g. `Embedding.t2Space`)
 * `Set.EqOn.closure`: If two functions are equal on some set `s`, they are equal on its closure.
 * `IsCompact.isClosed`: All compact sets are closed.
-* `locally_compact_of_compact_nhds`: If every point has a compact neighbourhood,
-  then the space is locally compact.
+* `WeaklyLocallyCompactSpace.locallyCompactSpace`: A weakly locally compact Hausdorff space
+  is locally compact.
```

Comment:
Could you remind here the definition of weakly locally compact space, for discoverability?